src/environments/environment.prod.ts

```diff
@@ -6,6 +6,7 @@ export const environment = {
   production: true,
   issuer: '',
   clientId: '',
+  trialName: ''
```

Comment:
You don't need to add this to the production environment.ts - the name is set as an environment variable during deployment.

Reply:
ng build --prod
The build fails expecting trialName variable
hence as a fix, added the same